XCUITests/FxScreenGraph.swift

```diff
@@ -99,11 +100,18 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
             }
 
             scene.tap(startBrowsingButton, to: NewTabScreen)
+            scene.tap(app.buttons["Sign in to Firefox"], to: FxSyncLogin)
         }
 
         i += 1
     }
 
+    map.createScene(FxSyncLogin) { scene in
+        scene.backAction = {
+            app.buttons["Cancel"].tap()
```

Comment:
Thanks! I think that would be the safer so far. I just commited new changes removing the sign up page references from FxScreenGraph